### PR TITLE
{AKS} Add UDR capabilities

### PR DESCRIPTION
**Pull request by gtracer**
**History Notes:**  
(Fill in the following template if multiple notes are needed, otherwise PR title will be used for history note.)

[Component Name 1] (BREAKING CHANGE:) (az command:) make some customer-facing change.  
[Component Name 2] (BREAKING CHANGE:) (az command:) make some customer-facing change.

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

**Comment by yungezz** [comment]
hi @fengzhou-msft  could you pls help to review? thanks.

File: src/azure-cli/azure/cli/command_modules/acs/_help.py [hunk 1]
```diff
@@ -260,6 +260,10 @@
     type: int
     short-summary: Load balancer idle timeout in minutes.
     long-summary: Desired idle timeout for load balancer outbound flows, default is 30 minutes. Please specify a value in the range of [4, 120].
+  - name: --outbound-type
```
Comment (yungezz) [inline src/azure-cli/azure/cli/command_modules/acs/_help.py]: there's a PR in extension repo revert the behavior, could you pls sync: https://github.com/Azure/azure-cli-extensions/pull/1405? @gtracer

**Comment by gtracer** [comment]
Gonna look into adding some integration tests, ideally in this PR or if we get too close to the cut-off then into another

File: src/azure-cli/azure/cli/command_modules/acs/_help.py (hunk 1 above)
Comment (gtracer) [reply src/azure-cli/azure/cli/command_modules/acs/_help.py]: @yungezz I own both PRs. 

In extension, outbound-type is already enabled. That PR just removes one piece of validation (network-plugin) and adds a vnetsubnet-id validation. No core behavior is being reverted. Please let me know what you think is specifically being reverted

Comment (gtracer) [reply src/azure-cli/azure/cli/command_modules/acs/_help.py]: @yungezz following up as I need to merge this PR before the cutoff for the release

**Pull request merged**